subprojects/build-cache-packaging/src/main/java/org/gradle/caching/internal/packaging/impl/TarBuildCacheEntryPacker.java

```diff
@@ -89,24 +87,23 @@
 
     private static final Charset ENCODING = StandardCharsets.UTF_8;
     private static final String METADATA_PATH = "METADATA";
+    private static final String MD5_PAX_HEADER = "MD5";
+    private static final HashCode METADATA_HASH = HashCode.fromInt(0);
```

Comment:
This won't work anymore. I think we should just omit the header for metadata. That'd save some space, too.